src/enftun.c

```diff
@@ -230,9 +234,11 @@ enftun_connect(struct enftun_context* ctx)
     }
 
 #ifndef USE_PSOCK
-    if ((rc = enftun_conn_state_prepare(&ctx->conn_state, &ctx->loop,
-                                        trigger_reconnect, (void*) ctx,
-                                        ctx->config.fwmark)) < 0)
+    if ((rc = enftun_conn_state_prepare(
+                  &ctx->conn_state, &ctx->loop, trigger_reconnect, (void*) ctx,
+                  ctx->config.fwmark, &ctx->tlschan, &ctx->ipv6, &ip6_self,
```

Comment:
`ip6_self` has the correct address, has a misleading name.

Instead, let's create a new constant `ip6_enf_router` in `ip.h,c` with the same value and use it instead.  Add a comment stating that `enftun` and the ENF router share the same IP since they split the duties of a traditional IPv6 router.